### DISCUSSION TITLE
Can't make network requests on iPhone on Android devices

### DISCUSSION THREAD

**Discussion by D-Marc1** [ionic-team/capacitor#3543] [Q&A] [answered]
It works perfectly on my computer and even on the simulators. However, as soon as I make those same network requests on my iPhone for instance, I get messages like the following.

> Fetch API cannot load http://0.0.0.0:3333/login due to access control checks.

and 

> Failed to load resource: Could not connect to the server.

**Comment by jcesarmobile** [comment, marked as answer]
0.0.0.0 is an address on the machine calling that address.

On your computer it works because you have the server running on the same computer.

When you run on Android or iOS device, you don't have a server running on them, so it fails.

Use the local IP of the computer instead, something like 192.168.1.XX

**Reply by D-Marc1** [reply]
Wow, I feel stupid for asking this question in retrospect...

Thanks for the help!
